src/general/replicalimits/constraint.tmpl

```diff
@@ -4,7 +4,7 @@ metadata:
   name: k8sreplicalimits
   annotations:
     metadata.gatekeeper.sh/title: "Replica Limits"
-    metadata.gatekeeper.sh/version: 1.0.1
+    metadata.gatekeeper.sh/version: 1.1.0
```

Comment:
I would interpret the template specific "rego fix" as a [patch version bump](https://github.com/open-policy-agent/gatekeeper-library#new-policy).  The constraint changes don't apply to the template versioning.  Otherwise everything looks good and I'll LGTM once updated.

```suggestion
    metadata.gatekeeper.sh/version: 1.0.2
```